web/src/lib/utils.ts

```diff
@@ -275,6 +275,13 @@ export const oauth = {
     }
     return false;
   },
+  isAutoLaunchEnabled: (location: Location) => {
+    const value = 'autoLaunch=1';
+    if (location.search.includes(value)) {
+      return true;
+    }
+    return false;
```

Comment:
This could be a single line/satement

Reply:
It could but that would break the convention set by the statement above it:

```javascript

  isAutoLaunchDisabled: (location: Location) => {
    const values = ['autoLaunch=0', 'password=1', 'password=true'];
    for (const value of values) {
      if (location.search.includes(value)) {
        return true;
      }
    }
    return false;
  },
  isAutoLaunchEnabled: (location: Location) => {
    const value = 'autoLaunch=1';
    if (location.search.includes(value)) {
      return true;
    }
    return false;
  },
```

I can change both to single statements for consistency and brevity's sake.

Reply:
I didn't mean you should remove curly braces. They are required. I meant just return the expression like Daniel suggested